docs/glossary.md

```diff
@@ -0,0 +1,168 @@
+ Glossary of Technical Terms
+---
+
+## A
+- [Account Aliases](#account-aliases)
+- [Admin Transaction](#admin-transaction)
+- [Auto Associations](#asynchronous)
+- [Auto-Renew Accounts](#auto-renew-accounts)
+- [Address Book](#address-book)
+- [Application Accounts](#application-accounts)
+
+## B
+- [BLS](#bls)
+- [Block Proof](#block-proof)
+- [Block Signature](#block-signature)
+- [Block Stream](#block-stream)
+
+## C
+- [Civilian Accounts](#civilian-accounts)
+- [Config.txt](#config.txt)
+- [Congestion Pricing](#congestion-pricing)
+- [Consensus Time](#consensus-time)
+
+## D
+- [Delegate Call](#delegate-call)
+
+## E
+- [EOA](#eoa)
+
+## H
+- [Hollow Accounts](#hollow-accounts)
+
+## L
+- [Leaky Tests](#leaky-tests)
+
+## P
+- [Precompiles](#precompiles)
+
+## S
+- [Security Model V2](#security-model-v2)
+- [Sidecars](#sidecars)
+- [Synthetic Transaction](#synthetic-transactions)
+- [System Contracts](#system-contracts)
+- [System Transaction](#system-transaction)
+
+## T
+- [Token Allowance](#token-allowance)
+
+## W
+- [Weibar](#weibar)
+
+---
+
+## Address Book
+**Address Book**: The _**network**_ address book is data structure that contains information (such as node ID, alias, IP address, and weight) for all nodes in the network, 
+not just consensus, but also block nodes, future validator nodes and more.
+The address book is stored as a file on disk at the moment but will be stored purely in state as part of Dynamic Address Book phase 3.	Disambiguation: There is also an "address book" in files 0.0.101 and 0.0.102 which is different to the "network address book" which are manually managed by devops.
+
+
+## Account Aliases
+**Account Aliases**: These are identifiers which are public key values, and are mostly 33 byte EVM address values. The "triplet" form (e.g. 0.0.18273892) is the simpler and easier to "type".
+An alias is not intended to be user-friendly, but instead to offer a smart-contract friendly identifier. See https://hips.hedera.com/hip/hip-32 and https://hips.hedera.com/hip/hip-583 for more information.
+
+
+## Admin Transaction 
+**Admin Transaction**: A HAPI transaction which has been signed by the council, for example a Freeze Transaction
+
```

Comment:
Suggestion:

A HAPI transaction requires the transaction fee payer to be a privileged system account authorized to handle admin transactions. A privileged system account is defined as an account with an entity ID of 0.0.1000 or lower. Each type of admin transaction specifies which accounts can serve as authorized transaction fee payers. Regular users in the community are not permitted to authorize these types of transactions. Some privileged accounts, like account 0.0.2 or 0.0.58, are owned by the Hedera Council. This means their keys are on the account and their signatures would be required if the account is being modified or used to pay for admin transactions. 

Examples of Admin Transactions include the following:
- Freeze Transactions
- System Delete
- System Undelete 
- Address Book File 0.0.101/102 updates
- Exchange Rate File 0.0.111 updates
- Dynamic Properties File 0.0.121 updates

Not sure if this is too much. Feel free to edit as you see fit.